core/primitives/src/runtime/parameter_table.rs

```diff
@@ -242,3 +243,209 @@ impl FromParameterTable for RuntimeFeesConfig {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::{InvalidConfigError, ParameterTable};
+    use near_primitives_core::parameter::Parameter;
+    use std::collections::BTreeMap;
+
+    #[track_caller]
+    fn check_parameter_table(base_config: &str, diffs: &[&str], expected: &ParameterTable) {
```

Comment:
This would probably be a little more straightforward to use with a signature like this:

```suggestion
    fn check_parameter_table(base_config: &str, diffs: &[&str], expected: impl IntoIterator<Item = (Parameter, &str)>) {
```

and then created the btreemap or whatever other structures to compare with internally.

Then the callers could look something like this:

```rust
        check_parameter_table(
            base_0,
            &[],
            [
                (Parameter::RegistrarAccountId, "registrar"),
                (Parameter::MinAllowedTopLevelAccountLength, "32"),
                (Parameter::StorageAmountPerByte, "100000000000000000000"),
                (Parameter::StorageNumBytesAccount, "100"),
                (Parameter::StorageNumExtraBytesRecord, "40"),
            ],
        );
```

Reply:
Yes, much cleaner! Done as you suggested